claim/README.md

```diff
@@ -1,41 +1,223 @@
-common invitation mechanism, which begins with the administrators of the workspace creating a **claiming-token**. They,
-or other users is their organization, can then use the claiming-token to add an agent to their workspace.
+Agent claiming is part of the onboarding process when creating a Space in Netdata Cloud. A Space's administrator creates
+a **claiming token**, which is used to add an Agent to their Space via the [Agent-Cloud link (ACLK)](/aclk/README.md).
 
-To claim a Netdata agent, you first send a claiming request to Netdata Cloud (from the agent node). Once the
-Netdata Cloud validates the claiming request of the agent (based on the claiming token), and returns a successful
-result, the node is considered claimed.
+Claiming nodes is a security feature in Netdata Cloud. Through the process of claiming, you demonstrate in a few ways
+that you have administrative access to that node and the configuration settings for its Agent. By logging into the node,
+you prove you have access, and by using the claiming script or the Netdata command line, you prove you have write access
+and administrative privileges.
 
-## Claiming script
+Only the administrators of a Space in Netdata Cloud can view the claiming token and accompanying script generated by
+Netdata Cloud.
 
-The user can claim an agent by directly calling the `netdata-claim.sh` script **as the netdata user** and passing the
-following arguments:
+> The claiming process ensures no third party can add your node, and then view your node's metrics, in a Cloud account,
+> Space, or War Room that you did not authorize.
+
+By claiming a node, you opt-in to sending data from your Agent to Netdata Cloud via the ACLK. This data is encrypted by
+the RSA keypair created during claiming, sent to Cloud, and then redirected your browser with TLS encryption. While the
+data does flow through Netdata Cloud servers on its way from Agents to the browser, we do not store or log it.
+
+## How to claim a node
+
+You can claim a node during the Cloud onboarding process, or after you created a Space by clicking on the **USER's
+Space** dropdown, then **Manage Claimed Nodes**.
+
+> Only the administrators of a Space in Netdata Cloud can view the claiming token and accompanying script generated by
+> Netdata Cloud.
+
+To claim a Node, copy the script given by Cloud and paste it into your node's terminal. The command looks similar to the
+following:
+
+```bash
+netdata-claim.sh -token=TOKEN -rooms=ROOM1,ROOM2
+```
+
+Hit **Enter**. The script should return `Agent was successfully claimed.`.
+
+> Your node may need up to 60 seconds to connect to Netdata Cloud after finishing the claiming process. Please be
+> patient!
+
+If the claiming script returns errors, see the [troubleshooting information](#troubleshooting).
+
+### Claiming through a proxy
+
+A Space's administrator can claim a node through a SOCKS5 (preferred) or HTTP(S) proxy.
```

Comment:
I would remove "(preferred)" because we support both and it sounds as if we prefer proxied connections to direct, which I don't know any reason for.